notebooks/0.download-data/0.download-data.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# activate conda env
+conda activate cell-injury
+
+# convert notebooks into python scripts
+jupyter nbconvert --to python --output-dir=nbconverted/ *.ipynb
```

Comment:
personal preference here but I like using `scripts` instead of `nbconverted` if running from shell